docs/guides/getting-started/prerequisites.md

```diff
@@ -47,6 +47,17 @@ Alternatively, you could use `winget` to install rustup using the following comm
 winget install --id Rustlang.Rustup
 ```
 
+:::caution MSVC toolchain as default
+
+For full support for Tauri and tools like [`trunk`] make sure the MSVC Rust toolchain is the selected `default host triple` in the installer dialog.<br/>Depending on your system it should be either `x86_64-pc-windows-msvc`, `i686-pc-windows-msvc`, or `aarch64-pc-windows-msvc`.
+
+If you already had Rust installed, you can make sure the correct toolchain is installed by running this command:
```

Comment:
```suggestion
If you already have Rust installed, you can make sure the correct toolchain is installed by running this command:
```

Reply:
i wanted to make clear that this means, "if you had it installed prior to reading this guide". Does this still come across using `have`? (i should've worked harder in school 😞)

Reply:
They convey the same thing, just different tenses. "have" is present tense (would which we prefer overall for writing style). Since you have "already" in front of it then that conveys it was installed prior to reading.

Reply:
Okay, thank you